v2/tools/generator/internal/astmodel/types.go

```diff
@@ -361,25 +416,6 @@ func (set TypeDefinitionSet) ResolveResourceSpecAndStatus(resourceDef TypeDefini
 	}, nil
 }
 
-// Process applies a func to transform all members of this set of type definitions, returning a new set of type
-// definitions containing the results of the transformation, or possibly an error
-// Only definitions returned by the func will be included in the results of the function. The func may return a nil
-// TypeDefinition if it doesn't want to include anything in the output set.
-func (set TypeDefinitionSet) Process(transformation func(definition TypeDefinition) (*TypeDefinition, error)) (TypeDefinitionSet, error) {
```

Comment:
I removed this because I observed there seemed to be no usage of it. If folks think I should keep it happy to though.

Reply:
We moved to a different idiom; happy for this to go.